wemake_python_styleguide/visitors/ast/operators.py

```diff
@@ -268,3 +269,35 @@ def visit_NamedExpr(
         """
         self.add_violation(consistency.WalrusViolation(node))
         self.generic_visit(node)
+
+
+@final
+class BitwiseOpVisitor(base.BaseNodeVisitor):
+    """Checks bitwise operations are used correctly."""
+
+    def visit_BinOp(self, node: ast.BinOp) -> None:
+        """
+        Finds bad usage of bitwise operation with binary operation.
+
+        Raises:
+            BitwiseAndBooleanMixupViolation
+
+        """
+        self._check_BitOp(node)
```

Comment:
Please, name it logically, not structurely. 

For example: `_check_logical_bitwise_operator`